incubator/mssql-linux/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "mssql.fullname" . }}
+  labels:
+    app: {{ template "mssql.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: 1433
+      targetPort: 1433
+      protocol: TCP
+      name: {{ .Values.service.name }}
```

Comment:
This is not the service name, it's the port name. It probably doesn't make sense to make it configurable. I'd just hardcode it.

Reply:
I went ahead and removed this all together.